src/Expand.hs

```diff
@@ -201,8 +201,24 @@ expand eval ctx xobj =
               Right (XObj (Lst [XObj Macro _ _, _, XObj (Arr _) _ _, _]) _ _) ->
                 --trace ("Found macro: " ++ pretty xobj ++ " at " ++ prettyInfoFromXObj xobj)
                 eval ctx'' xobj
-              Right (XObj (Lst [XObj (Command callback) _ _, _]) _ _) ->
-                getCommand callback ctx args
+              Right (XObj (Lst [XObj (Command (NullaryCommandFunction nullary)) _ _, _, _]) _ _) ->
+                nullary ctx''
+              Right (XObj (Lst [XObj (Command (UnaryCommandFunction unary)) _ _, _, _]) _ _) ->
+                case expandedArgs of
```

Comment:
Should these use `args` or `expandedArgs`?

Reply:
Not sure, do you think it ever matters? I'll merge as is, seems fine...